src/pages/Acknowledgements/pageContent.tsx

```diff
@@ -1,193 +1,289 @@
+      />
     ),
-    acknowledgementText:
-      'I will safeguard my key-stores and give them to my validator software in order to start validating.',
   },
-  [AcknowledgementIdsEnum.transferDelay]: {
-    title: 'Transfer delay',
+  [AcknowledgementIdsEnum.commitment]: {
+    title: (
+      <FormattedMessage defaultMessage="Validating is a long-term commitment" />
+    ),
     content: (
       <>
-        <Text size="large" className="my10">
-          Transfers between validators are disabled for now. Validators will
-          have to wait until <i>the merge</i> (around two years) to be able to
-          withdraw their ${TICKER_NAME}.
+        <Text size="medium" className="my10">
+          <FormattedMessage
+            defaultMessage={`Transfers between validators aren't possible today. You will have to
+              wait until mainnet merges with the Beacon Chain (around two years) before you can
+              withdraw your {TICKER_NAME}.`}
+            values={{ TICKER_NAME }}
+          />
         </Text>
         <Link
-          external
-          to="https://ethereum.org/en/eth2/beacon-chain/#relationship-between-upgrades"
+          to="https://ethereum.org/en/eth2/docking"
           className="my10"
           primary
-          withArrow
         >
-          Learn More about the eth2 upgrades
+          <FormattedMessage defaultMessage="More on the merge" />
         </Link>
+        <Text size="medium" className="my10">
+          <FormattedMessage
+            defaultMessage="With transfers disabled for now, you won't be able to voluntarily exit
+            and then restart later. This means you need to be in it for the long haul."
+          />
+        </Text>
       </>
     ),
-    acknowledgementText: `I understand that I CAN NOT TRANSFER my staked ${TICKER_NAME} for a while, and I CAN NOT WITHDRAW until the merge.`,
+    acknowledgementText: (
+      <FormattedMessage
+        defaultMessage="I understand that I {cannotTransfer} my stake for a while, and I
+          {cannotWithdraw} until the merge. I understand that if I exit, I will not be able
+          to rejoin until much later. This is a long term commitment."
+        values={{
+          cannotTransfer: <BoldCaps>cannot transfer</BoldCaps>,
+          cannotWithdraw: <BoldCaps>cannot withdraw</BoldCaps>,
+        }}
+      />
+    ),
   },
-  [AcknowledgementIdsEnum.commitment]: {
-    title: 'Long-term commitment',
+  [AcknowledgementIdsEnum.earlyAdoptionRisks]: {
+    title: <FormattedMessage defaultMessage="Early adopter risks" />,
     content: (
-      <Text size="large" className="my10">
-        With transfers disabled for now, there’s no way for a validator to
-        voluntarily exit and then restart later. This means validators need to
-        be in it for the long haul.
+      <Text size="medium" className="my10">
+        <FormattedMessage
+          defaultMessage="You're joining a young network. As with any new piece of software,
```

Comment:
IMO, "young" personifies the network too much, I think we should go with another option. What about:
```suggestion
          defaultMessage="You're joining a novel network. As with any new piece of software,
```

Reply:
Felt "novel" doesn't clearly convey the point here, which is that the network is still very new and could be buggy... tried using "You're joining an emerging, growing network." Let me know if you disagree

Reply:
I don't think "emerging, growing" captures it either. The network has already emerged and eth1 is growing too. Some other options:
* pioneering
* trailblazing
* unprecedented

Reply:
I don't think "emerging, growing" captures it either. The network has already emerged and eth1 is growing too. Some other options:
* pioneering
* trailblazing
* unprecedented

Reply:
I think unprecedented works. Or we could reword it to "You're joining a network in its early stages." or "You're joining a type of network that's never existed before".

Reply:
I'm happy with either of those options.